scalafix-interfaces/src/main/java/scalafix/interfaces/ScalafixFileEvaluation.java

```diff
@@ -18,10 +18,21 @@
 
     /**
      *
-     * @return boolean true if there is no error when scalafix has been evaluated on the file
+     * @return boolean true if there is no error except LintError when scalafix has been evaluated on the file
      */
     boolean isSuccessful();
 
+    /**
+     *
+     * @return a more detailed error in case of scalafix ParseError, StaleSemanticdbError,
+     * MissingSemanticdbError or an UnexpectedError. LinterErrors are found in {@link ScalafixDiagnostic}
```

Comment:
I am not in front of an IDE, but looking at the code, it looks like the error message is only defined for `UnexpectedError`.

https://github.com/scalacenter/scalafix/blob/e31e5c946a14bb3839ef5c087b660953a3485db8/scalafix-cli/src/main/scala/scalafix/internal/v1/MainOps.scala#L122-L125

Reply:
true, but this has been fixed in this PR. 
```
case Failure(exception) =>
   .from(file,  ExitStatus.from(exception), exception.getMessage)
...

  def from(throwable: Throwable): (ExitStatus) =
    throwable match {
      case _: ParseException =>
        ExitStatus.ParseError
      case _: SemanticDocument.Error.MissingSemanticdb =>
        ExitStatus.MissingSemanticdbError
      case _: StaleSemanticDB =>
        ExitStatus.StaleSemanticdbError
      case NonFatal(_) =>
        ExitStatus.UnexpectedError
    }
````